Dockerfile.ci

```diff
@@ -23,45 +22,42 @@ RUN set -eux; \
         zlib1g-dev \
         libxxhash-dev \
         build-essential \
-        pkg-config \
-    ; \
-    dpkgArch="$(dpkg --print-architecture)"; \
+        pkg-config && \
+    dpkgArch="$(dpkg --print-architecture)" && \
     case "${dpkgArch##*-}" in \
         amd64) rustArch='x86_64-unknown-linux-gnu'; rustupSha256='3dc5ef50861ee18657f9db2eeb7392f9c2a6c95c90ab41e45ab4ca71476b4338' ;; \
         armhf) rustArch='armv7-unknown-linux-gnueabihf'; rustupSha256='67777ac3bc17277102f2ed73fd5f14c51f4ca5963adadf7f174adf4ebc38747b' ;; \
         arm64) rustArch='aarch64-unknown-linux-gnu'; rustupSha256='32a1532f7cef072a667bac53f1a5542c99666c4071af0c9549795bbdb2069ec1' ;; \
         i386) rustArch='i686-unknown-linux-gnu'; rustupSha256='e50d1deb99048bc5782a0200aa33e4eea70747d49dffdc9d06812fd22a372515' ;; \
         *) echo >&2 "unsupported architecture: ${dpkgArch}"; exit 1 ;; \
-    esac; \
-    url="https://static.rust-lang.org/rustup/archive/1.24.3/${rustArch}/rustup-init"; \
-    wget "$url"; \
-    echo "${rustupSha256} *rustup-init" | sha256sum -c -; \
-    chmod +x rustup-init; \
-    ./rustup-init -y --no-modify-path --profile minimal --default-toolchain $RUST_VERSION --default-host ${rustArch}; \
-    rm rustup-init; \
-    chmod -R a+w $RUSTUP_HOME $CARGO_HOME; \
+    esac && \
+    url="https://static.rust-lang.org/rustup/archive/1.24.3/${rustArch}/rustup-init" && \
```

Comment:
Why use && instead of ;?

Reply:
it's best practice
In case with ; (Semi-colon) operator - the execution of the second command is independent of the exit status of the first command. If the first command does not get successfully executed, then also the second command will get executed.

Reply:
set is redundant, must be removed

Reply:
Good to know. Thank you for clarifying.